whichone/static/js/main.js

```diff
@@ -44,6 +44,14 @@ $(document).ready(function () {
         }
     });
 
+    $('#feedback-icon').on('click', function (event) {
+        window.location.href = 'form';
+    });
+
+    $('#go-to-play').on('click', function (event) {
+        window.location.href = 'play';
+    });
```

Comment:
Are you able to use `anchor` elements to do the same thing?

Reply:
is it better practice? We can change all of our jqueries to anchor elements (I believe), what's the benefit?

Reply:
I guess my thought would be, why pollute JS with template level logic.

If we were using a framework or had some logic around the routing we might need to go through JS. 
The benefit of utilizing DOM features like this is:
* clean JS
* we get the little preview of the link in bottom left hand corner 😄 as a user I prefer to see where a link will take me before I navigate

Reply:
Thanks for the info! I've now changed it to use anchor elements